src/test/java/org/minions/devfund/katerinaanzoleaga/WhackAMoleTest.java

```diff
@@ -0,0 +1,125 @@
+package org.minions.devfund.katerinaanzoleaga;
+/**
+ * WhackAMole game tests.
+ */
+
+//import static org.junit.Assert.assertEquals;
+
+//import com.sun.org.apache.xpath.internal.operations.Equals;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)